app/javascript/listings/__tests__/ListingDashboard.test.jsx

```diff
@@ -180,7 +183,14 @@ const listings = {
   selectedListings: 'user',
 };
 
+/* eslint-disable no-unused-vars */
+/* global globalThis timestampToLocalDateTimeLong timestampToLocalDateTimeShort */
+
 describe('<ListingDashboard />', () => {
+  afterAll(() => {
+    delete globalThis.timeAgo;
```

Comment:
I think this is a copy paste error from the timeAgo test file?

Reply:
Oops, Sorry I though I did something wrong but forgot where I did it.